OuterWildsRandomSpeedrun/OuterWildsRandomSpeedrun.cs

```diff
@@ -209,6 +219,18 @@ private void ResetRunButton_OnClick()
             ModHelper.Menus.PauseMenu.Close();
         }
 
+        private void PingusModeButton_OnClick()
+        {
+            if (_spawnPointSelectorManager == null)
+            {
+                _spawnPointSelectorManager = new GameObject("SpawnPointSelectorManager");
+                var manager = _spawnPointSelectorManager.AddComponent<SpawnPointSelectorManager>();
+                manager.ModHelper = ModHelper;
+            }
+
+            _spawnPointSelectorManager.GetComponent<SpawnPointSelectorManager>().DisplayMenu();
```

Comment:
should we consider moving the `var manager`  to this scope so we don't need to getComponent again?

Reply:
I wanted this, but the line with `var manager` is adding the component and not getting it. I'm treating `_spawnPointSelectorManager` as a singleton, so the manager is only created once. I could fix it by introducing a new class field for `manager`, but then we end up with more fields. Let me know your preference.

Reply:
could just do 
`SpawnPointSelectorManager manager;`
above the if, and remove var?

Reply:
Did what we discussed over Discord